ui/common/src/userComplete.ts

```diff
@@ -1,24 +1,24 @@
-import * as xhr from 'common/xhr';
+import * as xhr from './xhr';
 import debounce from 'debounce-promise';
 
-export interface Result {
+export interface UserCompleteResult {
   result: LightUserOnline[];
 }
 
-interface Opts {
+export interface UserCompleteOpts {
   input: HTMLInputElement;
   tag?: 'a' | 'span';
   minLength?: number;
-  populate?: (result: LightUserOnline) => string;
-  onSelect?: (result: LightUserOnline) => void;
+  populate?: (result: LightUser) => string;
```

Comment:
why no `LightUserOnline` anymore?

Reply:
Because UserCompleteOpts was previously declared multiple times in different places. I picked [this version](https://github.com/lichess-org/lila/blob/d2e6b1484968822f11e5720a4f8c82af6fdac292/ui/%40types/lichess/index.d.ts#L111) but it works either way.